tests/test_box_Box.py

```diff
@@ -439,6 +439,49 @@ def test_plot_2d(self):
         box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
         box.plot()
 
+    def test_compute_distances_2d(self):
+        box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
+        testpoints = np.array([[-2.2, -1.3, 0], [0., 0., 0.], [0., 0., 0.]])
+        testpoints_query = np.array(
+            [[-0.5, -1.3, 0], [0.2, 0., 0.], [0., -0.4, 0.]])
+        distance = box.compute_distances(testpoints, testpoints_query)
+        npt.assert_allclose(distance, [0.3, 0.2, 0.4], rtol=1e-6)
+
+        # 1 dimensional array
+        distance = box.compute_distances(testpoints[0], testpoints_query[0])
+        npt.assert_allclose(distance, 0.3, rtol=1e-6)
+
+        with self.assertRaises(ValueError):
+            box.compute_distances(testpoints[:-1], testpoints_query)
+
+    def test_compute_distances_3d(self):
+        box = freud.box.Box(2, 3, 4, 1, 0, 0)
+        testpoints = np.array([[-2.2, -1.3, 2], [0., 0., 1.], [0., 0., 0.1]])
+        testpoints_query = np.array(
+            [[-0.5, -1.3, 2.], [0.2, 0., 1.], [0., -0.4, 0.1]])
+        distance = box.compute_distances(testpoints, testpoints_query)
+        npt.assert_allclose(distance, [0.3, 0.2, 0.4], rtol=1e-6)
+
+    def test_compute_all_distances_2d(self):
+        box = freud.box.Box(2, 3, 0, 1, 0, 0, is2D=True)
+        testpoints = np.array([[0., 0., 0.]])
```

Comment:
Just use the names `points` and `query_points` for these arrays.

Reply:
I simplified the variable names. I originally chose `testpoints` and `testpoints_query` for consistency with some previous tests, ie)
https://github.com/glotzerlab/freud/blob/1e99ce9e05113514725db1def1cecbfacea5cd60/tests/test_box_Box.py#L85-L92
I can change these test variables as well if this type of consistency is important.

Reply:
Overall I prefer `points` over `testpoints`. It's obviously a test. And `testpoints_query` is long and awka-awkward because they're normally called `query_points`, not `points_query`.